### PR TITLE
Add notification indicator to PR list items

### PR DESCRIPTION
**Pull request by txsmith**
screenshot from 2016-05-04 17 17 49

### CODE REVIEW

File: src/projects/project-list-item.html [hunk 1]
```diff
@@ -6,18 +6,14 @@
 <dom-module id="project-list-item">
   <template>
     <style>
-      .indicator-true {
+      .margin.indicator.true {
```
Comment (TimvdLippe) [inline src/projects/project-list-item.html]: I don't think `.margin` is required here.

File: src/projects/project-pull-request-list-item.html [hunk 2]
```diff
@@ -8,35 +8,44 @@
 <dom-module id="project-pull-request-list-item">
   <template>
     <style>
-      .ci-status-failed {
+      .margin.indicator.true {
```
Comment (TimvdLippe) [inline src/projects/project-pull-request-list-item.html]: Same as above.

File: src/projects/project-pull-request-list-item.html [hunk 3]
```diff
@@ -8,35 +8,44 @@
...
+      .icon-failed {
+        fill: var(--primary-action-red);
+      }
+      .icon-failed.passed {
         display: none;
       }
-      .ci-status-passed {
-        background-color: var(--primary-action-green);
+      .icon-passed {
+        fill: var(--pr-status-color);
```
Comment (TimvdLippe) [inline src/projects/project-pull-request-list-item.html]: I think this should be green instead. It is now gray, I prefer green to indicate a good CI.

**Comment by eanker** [comment]
lgtm

**Pull request merged**